gpflow/utilities/model_utils.py

```diff
@@ -1,9 +1,29 @@
+from typing import Any, Callable
+
 import tensorflow as tf
 
 from ..base import TensorType
 from ..experimental.check_shapes import check_shapes
 
 
+def assert_params_false(
```

Comment:
Nice. (I'm automatically trying to generalise this, but there's definitely no need at the moment.)

Reply:
> Nice. (I'm automatically trying to generalise this, but there's definitely no need at the moment.)

Yeah, I did spend some trying to think of more clever ways to do this, but I'm worried about making it too complicated and creating problems with TensorFlow compilation.